cli/cmd/root.go

```diff
@@ -50,9 +55,9 @@ func NewCmdRoot() *cobra.Command {
 	if legacyTelemetry != "" {
 		defaultTelemetryValue = "none"
 	}
-	err := telemetryLevel.Set(env.GetEnvOrDefault("CQ_TELEMETRY_LEVEL", defaultTelemetryValue))
+	err = telemetryLevel.Set(env.GetEnvOrDefault("CQ_TELEMETRY_LEVEL", defaultTelemetryValue))
 	if err != nil {
-		fmt.Fprintf(os.Stderr, "failed to set telemetry level: "+err.Error())
+		fmt.Fprintf(os.Stderr, "failed to set telemetry level: %s", err.Error())
```

Comment:
```suggestion
		fmt.Fprintf(os.Stderr, "failed to set telemetry level: %v", err)
```

Reply:
fixed in [217cc13](https://github.com/cloudquery/cloudquery/pull/18878/commits/217cc1309b098c279c07aa4695cc7ca1ebd2eb0e)